2016/ideas-list-swc.md

```diff
@@ -55,25 +76,23 @@ task, but thanks to their API we can do this automatically in our application.
 
 ### Technical details
 
-Due to the fact that [AMY][] is a [Django][] application, it is suggested:
-
-* to have experience with [Django][] and [Python][]
-* to create the project as [Django application](https://docs.djangoproject.com/en/1.9/ref/applications/#projects-and-applications)
-
-This will allow for faster integration with AMY, if the there's time left in
-the Summer.
+Due to the fact that [AMY][] is a [Django][] application, it is suggested
+to have experience with [Django][] and [Python][]
+since create the project as [Django application](https://docs.djangoproject.com/en/1.9/ref/applications/#projects-and-applications)
+will allow faster integration with AMY and easy maintenance of the code.
 
 Project's interface will probably use one of many JavaScript plotting
-libraries, so it is required to research them and select one most promising.
+libraries, e.g. [D3js](http://d3js.org/) or [plotly.js](https://plot.ly/javascript/),
+so it is required to research them and select one most promising.
```

Comment:
I'd replace D3 with Metrics-Graphics (http://metricsgraphicsjs.org/) since the former is not a plotting library.